composer.json

```diff
@@ -50,7 +50,12 @@
   },
   "config": {
     "vendor-dir": ".Build/vendor",
-    "bin-dir": ".Build/bin"
+    "bin-dir": ".Build/bin",
```

Comment:
Is this needed? Typically it appears only in the main composer.json

Reply:
Yes, this is needed for the CI (see e.g. script area in composer.json) currently ☺️